Assets/Scripts/Game.cs

```diff
@@ -4,4 +4,15 @@
 
 public class Game : MonoBehaviour
 {
+    public GameObject CardPrefab;
+
+
+    void Start()
+    {
+        Instantiate(CardPrefab, new Vector3(0, 0, 0), Quaternion.identity);
+    }
```

Comment:
WDYT about using Awake method for Initiating functions ?

Reply:
Only for testing purpose